hack/update-deps.sh

```diff
@@ -10,7 +10,7 @@ set -o pipefail
 cd ${ROOT_DIR}
 
 # This controls the knative release version we track.
-KN_VERSION="release-0.18"
+KN_VERSION="release-0.19"
```

Comment:
Does this actually work if you run `./hack/update-deps.sh --upgrade`? It shouldn't. I believe we need to invest a little since upstream moved to `buoy` for floating the deps and that seems to break our current flow.

/hold

Reply:
Doesn't this than have "impact" to other kn libs?

For instance there is no `release-0.19` on eventing-contrib - that's on `knative-sandbox/eventing-kafka` (I was thinking to update that, when I actually can update that (e.g. we have our image builds for the new repo etc etc.)

Reply:
Yes it does, and yes that's one of the failures this causes.

You can theoretically update this right away to unblock this specific issue. No need for 0.19 images to land here usually.